src/main.rs

```diff
@@ -86,9 +87,7 @@ fn main() -> Result<()> {
     } else if !std::io::stdin().is_terminal() {
         Box::new(BufReader::new(stdin()))
     } else {
-        Opts::command()
-            .print_help()
-            .expect("Printing help should not fail");
+        println!("You must provide input via the --input flag or piped via stdin.");
```

Comment:
I kept the behaviour the same as before (i.e. print a message, but don't error). Should this actually returning an error (e.g. `anyhow!`)?

Reply:
I've updated this to use `anyhow!` and confirmed it returns a non-zero status. I also updated the (ignored) test for it.